### PR TITLE
fix(css): Improve contrast for angle

### PR DESCRIPTION
**Pull request by rohini-ranjanR**
### Description
improve contrast by changing color for angle issue #2054 

### Related issues and pull requests
Relates #2054

### CODE REVIEW

File: live-examples/css-examples/values-and-units/angle.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #example-element {
-    background-color: lightsalmon;
+    background-color: #ff5b9b;
```
Comment (NiedziolkaMichal) [inline live-examples/css-examples/values-and-units/angle.css]: This color still doesn't meet the minimum contrast guidelines, as the text to background contrast ratio is `2.91`.
How about the #0454aa - which meets [AAA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast#related_wcag_success_criteria) requirement with score `7.36`?

**Review by NiedziolkaMichal — CHANGES_REQUESTED** [review]

File: live-examples/css-examples/values-and-units/angle.css (hunk 1 above)
Comment (rohini-ranjanR) [reply live-examples/css-examples/values-and-units/angle.css]: I think this color looks good in dark theme but when we switch to light the text in black will little hard to look. What about #ffb703 it looks good in both dark and light theme.

Comment (NiedziolkaMichal) [reply live-examples/css-examples/values-and-units/angle.css]: This color has a contrast ratio of `1.75` and even for me it's difficult to read the content:
image

Comment (rohini-ranjanR) [reply live-examples/css-examples/values-and-units/angle.css]: ok suggested color applied

Comment (NiedziolkaMichal) [reply live-examples/css-examples/values-and-units/angle.css]: @rohini-ranjanR We could still go with something else that you think would be better for both light and dark themes. To check the contrast ratio, first, inspect `#example-element` in dev tools, then add `color: white;` declaration in it(this shouldn't have any effect), and now click on the `white` dot:
image

The contrast ratio will be displayed at the bottom, comparing color with `background-color`.

Comment (rohini-ranjanR) [reply live-examples/css-examples/values-and-units/angle.css]: Thanks @NiedziolkaMichal . I am using #0118f3 here that is giving a contrast of `8.65`

image

looks like this in **dark theme** 

image

and this in **light theme** 
image

Comment (NiedziolkaMichal) [reply live-examples/css-examples/values-and-units/angle.css]: Thanks for noticing. How about also changing the color to white, so the example looks good in both themes?

Comment (rohini-ranjanR) [reply live-examples/css-examples/values-and-units/angle.css]: okay, thanks for your suggestions.

**Comment by NiedziolkaMichal** [comment]
@rohini-ranjanR Could you fix the ESLint error, please?

**Review by NiedziolkaMichal — APPROVED** [review]
It looks good. Thank you for your contribution.

**Pull request merged**